README.md

```diff
@@ -1,129 +1,199 @@
+Upon declaration, all variables are automatically assigned the value of `undefined`. It's only after we assign a new value that the variable will contain something other than undefined. We can use the `typeof` operator to check the data type of the value currently stored in a variable:
+```js
+var language;
+//=> undefined
 
-In the browser, global variables are all properties of `window`. What is `window`? Well, it's the — erm — window in which the browser displays the current page. It holds a whole bunch of things (which is probably obvious), global variables among them.
+typeof language;
+//=> "undefined"
 
-Accidentally setting variables to "global" is a dangerous, imprecise coding mistake.
+language = "JavaScript";
+//=> "JavaScript"
 
-### `var`
+typeof language;
+//=> "string"
+```
 
-In the olden days (1995), JavaScript had one way to declare a non-global variable, `var`. Using the keyword `var` creates _local variable_, meaning that it is only accessible inside the current function in which it is declared. (However, this is a bit tricky - If 'var' is not declared *inside* a function, it's actually still global!)
+Once a variable has been created with `var`, we can reassign it to our heart's content:
+```js
+var pi = 3.14159;
+//=> undefined
 
-**Flat fact**: What's a function? Well, functions are ways of organizing our code so that we can run it repeatedly. We'll talk more about functions in a bit, but we need to mention them here because they're _vital_ to understanding variable scope in JavaScript (which we will tackle in greater depth when we cover functions). Don't worry too much right now about what exactly a function is; we just want to get used to seeing the word.
+typeof pi;
+//=> "number"
 
-Here's how this works — follow along in your console!
+pi = "the ratio between a circle's circumference and diameter";
+//=> "the ratio between a circle's circumference and diameter"
 
-``` javascript
-// declare the variable
-var word
+typeof pi;
+//=> "string"
+```
 
-// assign a value to the variable
-word = 'bird'
+The data that's stored in our variable might change over time, but at any moment we can retrieve its current contents:
+```js
+var language = "Mocha";
+//=> undefined
 
-console.log(word) // 'bird'
+language = "LiveScript";
+//=> "LiveScript";
 
-// assign another value to the variable
-word = 'dog'
+language = "JavaScript";
+//=> "JavaScript";
 
-console.log(word) // 'dog'
+language;
+//=> "JavaScript";
 ```
 
-Now we have declared a local variable `word`, and we can assign and reassign its value as we please.
-
-We can perform variable declaration and assignment on the same line to save space:
+## Beyond `var`
+As we alluded to earlier, there is almost no reason to use `var` in a post-ES2015 world. `var` comes with a ton of baggage in the form of scope issues, which we will discuss in the lesson on scope in JavaScript, and allowing developers to play a little too fast and loose with variable declarations.
```

Comment:
because "it's what [they] will encounter in any pre-ES2015 code", but yeah should probably reiterate that it's pretty ubiquitous